src/Interface/JSONParserInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php declare(strict_types=1);
+
+/*
+ Copyright (c) 2022, Manticore Software LTD (https://manticoresearch.com)
+
+ This program is free software; you can redistribute it and/or modify
+ it under the terms of the GNU General Public License version 2 or any later
+ version. You should have received a copy of the GPL license along with this
+ program; if you did not, you can find it at http://www.gnu.org/
+ */
+
+namespace Manticoresearch\Buddy\Interface;
+
+use \Iterable;
```

Comment:
Using `\` is not necessary. I suggest changing it to use Iterable for the sake of consistency. Actually, this is a good rule to include in the next code style's update